packages/react-native/ReactAndroid/src/main/jni/react/turbomodule/ReactCommon/BindingsInstallerHolder.cpp

```diff
@@ -11,12 +11,11 @@
 
 namespace facebook::react {
 
-BindingsInstallerHolder::BindingsInstallerHolder(
-    std::function<void(facebook::jsi::Runtime& runtime)> bindingsInstaller)
+BindingsInstallerHolder::BindingsInstallerHolder(BindingsInstallFunc bindingsInstaller)
     : bindingsInstaller_(std::move(bindingsInstaller)) {}
 
-void BindingsInstallerHolder::installBindings(jsi::Runtime& runtime) {
-  bindingsInstaller_(runtime);
+void BindingsInstallerHolder::installBindings(jsi::Runtime& runtime, std::shared_ptr<CallInvoker> callInvoker) {
+  bindingsInstaller_(runtime, callInvoker);
```

Comment:
There's no expectation that callInvoker will be copied here, so prefer passing it as a const ref

```suggestion
void BindingsInstallerHolder::installBindings(jsi::Runtime& runtime, const std::shared_ptr<CallInvoker>&  callInvoker) {
  bindingsInstaller_(runtime, callInvoker);
```